### PR TITLE
Drop SlimeEngine

### PR DESCRIPTION
**Pull request by timsueberkrueb**
Drop SlimeEngine and switch to using QtWebEngine only.
See #40.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[submodule "features"]
+	path = features
+	url = https://github.com/lirios/qmake-features
```
Comment (plfiorini) [inline .gitmodules]: use `git://github.com/lirios/qmake-features.git` since it's faster than https

**Review by plfiorini — CHANGES_REQUESTED** [review]
seems good but i haven't tried building it, i assume it does :)

**Review by plfiorini — APPROVED** [review]

**Comment by timsueberkrueb** [comment]
Huh, travis fails now.

**Comment by timsueberkrueb** [comment]
Damn, copied the wrong url, sorry for that.

**Comment by timsueberkrueb** [comment]
Okay, thanks @plfiorini. If we find any unnecessary/leftover QML wrapper code in the future, I think we can just simplify/fix it on `develop`.

**Comment by plfiorini** [comment]
@tim-sueberkrueb yep

**Pull request merged**